README.md

```diff
@@ -1,204 +1,176 @@
+  Find your username in [JFrog](https://digitalasset.jfrog.io/ui/admin/artifactory/user_profile); usually `firstname.lastname`.
+- `ARTIFACTORY_PASSWORD`: 
+  This is your **API Key**, *NOT* your password. Find your API key at the bottom of **Authentication Settings** in your profile.
 
-When working on the documentation, you'll want to see the results of your work
-locally before opening a pull request.
+## Build and view the docs locally
 
-To do that, switch to the `docs` directory in the repository:
+`cd` to the `docs` directory in the repository:
 
-```
+```zsh
 cd docs
 ```
 
-Download the documentation tarballs from the daml and canton repos:
+### Download docs
 
-```
+Run the following script to download the documentation tarballs from the Daml and Canton repos:
+
+```zsh
 ./download.sh
 ```
 
-## Preview
+### Live preview
 
-For local development, use the `live-preview.sh` script. The files for the
-unified documentation will be in the `docs/workdir/build` folder; changes to
-those files should be immediately reflected.
+Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Edits should be immediately reflected.
 
-Remember that once you are happy with your changes, you have to
-mirror them as PRs on the daml/canton repositories.
+:warning: If you make changes to docs managed by the other repos, you have to commit them there. You can only make changes to the top-level `index.rst` files, containing the master TOC, that live in this repository. 
 
-Only changes to the top-level `index.rst` files can be made directly in this
-repository.
+:warning: If you do make a change to the TOCs in this repo, you then have to duplicate the change in the relevant satellite repo.
 
-```
+```zsh
 ./live-preview.sh
 ```
 
-## Full build
+### Full build
 
-To view the PDF docs or the exact HTML files that we publish to
-[docs.daml.com](https://docs.daml.com), you can also run the full build
-locally:
+To build the PDF docs or the exact HTML files that we publish to [docs.daml.com](https://docs.daml.com), run the full build locally:
 
-```
+```zsh
 ./build.sh
 ```
 
-This leaves you with two files:
+This produces the following:
 
-```
+```zsh
 $ ls docs/workdir/target
 html-docs-2.0.0-snapshot.20211220.8736.0.040f1a93.tar.gz
 pdf-docs-2.0.0-snapshot.20211220.8736.0.040f1a93.pdf
 $
 ```
 
-To view the html docs, extract them and launch a webserver, e.g. via Python and
-point your browser at `http://localhost:8000`
+To view the html docs, extract them and launch a webserver, e.g. via Python, and point your browser at `http://localhost:8000`.
 
-```
+```zsh
 tar xf docs/workdir/target/html-docs-$(jq -r '.daml' LATEST).tar.gz
 cd html
 python3 -m http.server 8000 --bind 127.0.0.1
 ```
 
-## Setup
+## How to commit changes to the docs
 
-### direnv
+:warning: This repo ignores any pre-2.0.0 docs; those will still be published using their existing release process: tbc what that is!
```

Comment:
We've not even tried to change the docs for the 1.x site in over a year so unlikely to ever happen.

The process for 1.x is entirely encapsulated in the daml repo; I don't think it should be documented here.

Reply:
OK, I'll add something about reaching out to daml team if it ever comes up.